test/dlc_tests/container_tests/bin/healthcheck_tests/efa_checker_single_node.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+echo "Start EFA checker single node test"
+efa_checker_single_node --no_instance_id --verbose
+echo "Complete EFA checker single node test"
```

Comment:
Would need to rerun the tests due to the change in script name - I would modify that.

Reply:
Additionally, whenever we make a code change lets please ensure that the change executes on the test. For eg., the efa health checker sanity test is still commented out [link](https://github.com/aws/deep-learning-containers/blob/5c9d9db496da66334e896748a5f57d7b9c62085f/test/dlc_tests/sanity/test_health_check.py#L110) and has not been able to run the changes that were made on the script.

Also, as discussed, since we do not need to run this ([link](https://github.com/aws/deep-learning-containers/pull/3293/files#diff-f34999968b21c45bf8cec647c0ab71d16cfa107b056f2cad1a46ba363bc80662R118)) healthchecker sanity test on a heavy instance, lets make sure that we use a lighter instance type if possible.